### PR TITLE
Remove AT TIME ZONE 'UTC' in insert/update queries

### PR DESCRIPTION
**Pull request by mantasaudickas**
Related to #284

### CODE REVIEW

File: tests/Hangfire.PostgreSql.Tests/Utils/ConnectionUtils.cs [hunk 1]
```diff
@@ -16,7 +15,7 @@ public static class ConnectionUtils
     private const string DefaultDatabaseName = @"hangfire_tests";
     private const string DefaultSchemaName = @"hangfire";
 
-    private const string DefaultConnectionStringTemplate = @"Server=127.0.0.1;Port=5432;Database=postgres;User Id=postgres;Password=password;";
+    private const string DefaultConnectionStringTemplate = @"Server=127.0.0.1;Port=5432;Database=postgres;User Id=admin;Password=admin;";
```
Comment (azygis) [inline tests/Hangfire.PostgreSql.Tests/Utils/ConnectionUtils.cs:18]: Revert.

**Review by azygis — CHANGES_REQUESTED** [review]

File: tests/Hangfire.PostgreSql.Tests/Utils/ConnectionUtils.cs (hunk 1 above)
Comment (mantasaudickas) [reply tests/Hangfire.PostgreSql.Tests/Utils/ConnectionUtils.cs]: Done.

Comment (azygis) [reply tests/Hangfire.PostgreSql.Tests/Utils/ConnectionUtils.cs]: Close, not quite. Password should be password, not postgres.

Comment (mantasaudickas) [reply tests/Hangfire.PostgreSql.Tests/Utils/ConnectionUtils.cs]: Oh, you are right. Sorry.
Now should be fully reverted.

**Review by azygis — APPROVED** [review]

**Comment by azygis** [comment]
Thanks!

**Pull request merged**